src/index.html

```diff
@@ -1,17 +1,159 @@
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form method="post" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field" for="username">
+          Surname:
+          <input
+            type="text"
+            name="username"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form-field" for="name">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form-field" for="age">
+          How old are You?:
+          <input
+          type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </label>
+        <label class="form-field" for="age">
+          Full date of birth:
+          <input
+            type="date"
+            name="birthDate"
+          >
+        </label>
+        <label class="form-field" for="agreement">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="form-field" for="email">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            required
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form-field" for="password">
+          Password:
+          <input
+            type="password"
+            name="password"
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="yes"
+              value="Yes"
+            >
+              Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="no"
+              value="No"
+            >
+              No
+          </label>
+        </div>
+        <label class="form-field" for="favouriteColor">
+          What is your favourite color?
+          <input
+            type="color"
+            name="favouriteColor"
+          >
+        </label>
+        <label class="form-field" for="bedTime">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="bedTime"
+          >
+        </label>
+        <label class="form-field" for="favouriteCarBrands">
+          What are your favorite brand of cars?
+          <select
+            name="favouriteCarBrands"
+            id="favouriteCarBrands"
+            multiple>
```

Comment:
a tiny one, but quite important for readability
```suggestion
          <select
            name="favouriteCarBrands"
            id="favouriteCarBrands"
            multiple
          >
```